cnames_active.js

```diff
@@ -2493,7 +2494,7 @@ var cnames_active = {
   "simonhans": "simonhans.github.io",
   "simpldb": "hosting.gitbook.io", // noCF
   "simple": "lescinskiscom.github.io/simple-js",
-  "simple-pip": "sleepiie.github.io/simple-pip",
+  "simple-pip": "frogweezer.github.io/simple-pip",
```

Comment:
```suggestion
  "simple-pip": "sleepiie.github.io/simple-pip",
```
Will be handled by #6677